src/explore-education-statistics-frontend/src/components/AccordionWithAnalytics.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { logEvent } from '@frontend/services/googleAnalyticsService';
+import Accordion, { AccordionProps } from '@common/components/Accordion';
+
+const AccordionWithAnalytics = (props: AccordionProps) => {
+  return (
+    <Accordion
+      {...props}
+      analytics={(accordionHeading: { id: string; title: string }) => {
```

Comment:
No need to re-define the parameter type here, Typescript will already infer the correct type, so you can just do:
```javascript
onToggle={section => {...}}
```